aii-dhcp/src/main/perl/dhcp.pm

```diff
@@ -315,9 +320,18 @@ sub Configure
     my $opts = $config->getElement("/system/aii/dhcp")->getTree();
     my $tftpserver = "";
     my $additional = "";
+    my filename = "";
     if ($opts->{tftpserver}) {
         $tftpserver = $opts->{tftpserver};
     }
+    
+    if ($opts->(filename)) {
+        $filename = $opts->(filename);
+    }
+    if ($rescue eq 'rescue' && $opts->(rescue)) {
```

Comment:
Why it is not an `elsif`? It makes no point to set the filename to `$opts->(filename)` in rescue mode or did I miss something?

Reply:
There might be a case when there's 'filename' set, and 'rescue' not, also if both 'filename' and 'rescue' are set.
I wanted to make sure if there's a 'filename' set, get it anyway, and if there's 'rescue' there as well, overwrite it.